app/models/step_by_step_page.rb

```diff
@@ -86,7 +86,7 @@ def unpublished_changes?
   end
 
   def can_be_published?
-    has_draft? && !scheduled_for_publishing? && steps_have_content? && links_checked?
+    has_draft? && !scheduled_for_publishing? && steps_have_content? && links_checked_since_last_update?
```

Comment:
Nitpick: Can each condition be moved to a separate line to make it easier to read?

e.g.
```
condition_one? &&
  condition_two? &&
  condition_three?
```